src/Order.js

```diff
@@ -0,0 +1,18 @@
+import React, { useState } from "react";
+import RestaurantButton from "./RestaurantButton";
+
+function Order({ orderType }) {
+  const [orders, setOrders] = useState(0);
+
+  function orderOne() {
+    setOrders(currentCount => currentCount + 1);
```

Comment:
you don't need to pass a function as an argument to the setOrders function here, because the state type is a number which is immutable (i.e. a new number will always be returned from a calculation). So this can be simplified to
```
function orderOne() {
    setOrders(orders + 1);
}
```